### ISSUE TITLE
[HTML-Validation] div is not allowed under h4

### ISSUE THREAD

**Issue by endlacer** [MurhafSousli/ngx-gallery#646]
The html validation in our pipeline brought up this issue:
for eample [here](https://github.com/MurhafSousli/ngx-gallery/blob/c03dfbe894f3e0a760511194897d46e32b647db7/projects/ng-gallery/src/lib/templates/gallery-image.component.ts#L62) there is a div under a heading-element
see: https://html-validate.org/rules/element-permitted-content.html

Also a aria-label would be necessary for these headings, since a screenread cannot read the svg:
 https://html-validate.org/rules/empty-heading.html
